src/main/java/net/fabricmc/loader/metadata/AbstractModMetadata.java

```diff
@@ -50,6 +50,14 @@ public CustomValue getCustomValue(String key) {
 		return getCustomValues().get(key);
 	}
 
+	/**
+	 * Converts a custom value to a json element.
+	 * This will be removed in a future version when we remove the gson dependency.
+	 *
+	 * @param value the custom value
+	 * @return a json element
+	 */
+	@Deprecated
```

Comment:
Hmm, does this need deprecation since it's not in an API class?

Reply:
It's really just a marker for the future when gson gets fully dropped